blog/_posts/2020-07-15-Tetiana-Dadakova.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title: "Future blogpost about MABE"
+date: 2020-07-15
+author: Tetiana
+---
+
+# Genome change logging using range map  
+
+For more information about MABE (Modular Agent-Based Evolution platform) see [this introduction](https://szendejo.github.io/waves/blog/Team-MABE.html).
+
+## Genome Class
+
+### Naïve Implementation
+
+The naïve implementation of the genome is using a vector data structure from the standard library. The overwrite, insert and remove operations are, therefore, implemented using the standard library algorithms, specifically:
```

Comment:
I would change the second line to "In this naïve implementation, the overwrite, insert and remove operations are thus implemented using the following standard library algorithms:" just to make it a bit more clear